pkg/kubelet/metrics/metrics_test.go

```diff
@@ -72,6 +72,30 @@ func TestImagePullDurationMetric(t *testing.T) {
 	})
 }
 
+func TestGoroutinesMetric(t *testing.T) {
```

Comment:
This test is not testing anything ...? We should check for the expected values of the metrics to make this useful.